src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -512,6 +552,11 @@ def check_job_output_states(
             msg,
         )
 
+        if response_type == STATUS:
+            self._check_pop_last_checked(output_states, last_checked)
+        else:
+            self.assertNotIn("last_checked", output_states)
```

Comment:
Easiest way to test the `last_checked` field compatibly with existing test harnessing is to check it then delete it